app/views/threadreplies/_new.html.erb

```diff
@@ -1,4 +1,17 @@
 <%= form_for [reply.thread, reply] do |f| %>
   <%= render partial: "md_editor", locals: {name: "threadreply[content]", content: reply.content} %>
-  <p><%= f.submit "Reply#{ ' (Locked)' if reply.thread.locked? }", class: "btn blue" %></p>
-<% end %>
\ No newline at end of file
+  <% nec_msg = "" %>
+  <% forum = Forum.find(reply.thread.forum_id) %>
+  <% if forum.necro_length != nil %>
+    <% if Threadreply.where(forumthread: reply.thread).count != 0 %>
+      <% prevAgo = Threadreply.where(forumthread: reply.thread).order(:id).last.created_at  %>
+      <% if prevAgo <= forum.necro_length.days.ago.utc %>
+        <% nec_msg = "You may be necroposting, as the last reply was made at least #{forum.necro_length} days ago. If you still wish to make this reply, press 'Ok'." %>
```

Comment:
No, because 1. The user can see that from the timestamp on the reply itself, and 2. Telling the exact number of days will not say the minimum time required for a warning, and that information is more beneficial to the user than the actual number of days (due to no. 1).